ghost_state_machine_node:
  ros__parameters:
    max_linear_vel: 1.5              # m/s
    max_steering_angular_vel: 16.75  # rad/s

    left_wheel_x:   0.126365
    left_wheel_y:   0.180975
    right_wheel_x:  0.126365
    right_wheel_y:  -0.180975
    back_wheel_x:   -0.187
    back_wheel_y:   0.0

    steering_kp: 0.2
    turret_kp: 0.2

    max_motor_rpm_true: 625.0